sources/api/apiclient/src/main.rs

```diff
@@ -715,6 +784,9 @@ mod error {
         #[snafu(display("Failed to exec: {}", source))]
         Exec { source: exec::Error },
 
+        #[snafu(display("Failed to get settings: {}", source))]
```

Comment:
We won't always be getting settings right?  Since you can get services, etc?

Reply:
I have a hard time with terminology in places like this.  Users are almost always going to use this to retrieve settings, since the other types are more "under the hood."  "get model" isn't intuitive... I'm not sure what to say, but I agree it could use a tweak...

Reply:
Yeah, agreed.  Leaving the text as "settings" probably isn't harmful since you're right, that's the majority use case.  I stewed for a bit and didn't come up with anything better (only really general and unhelpful ideas like "info").  I'll resolve this comment and move on... :)